src/builders/nodejs/granular/default.nix

```diff
@@ -7,65 +7,64 @@
   runCommand,
   stdenv,
   writeText,
-
   # dream2nix inputs
   builders,
   externals,
   utils,
   ...
 }:
-
 {
   # Funcs
-
   # AttrSet -> Bool) -> AttrSet -> [x]
-  getCyclicDependencies,        # name: version: -> [ {name=; version=; } ]
-  getDependencies,              # name: version: -> [ {name=; version=; } ]
-  getSource,                    # name: version: -> store-path
-  buildPackageWithOtherBuilder, # { builder, name, version }: -> drv
-
+  getCyclicDependencies,
+  # name: version: -> [ {name=; version=; } ]
+  getDependencies,
+  # name: version: -> [ {name=; version=; } ]
+  getSource,
+  # name: version: -> store-path
+  buildPackageWithOtherBuilder,
+  # { builder, name, version }: -> drv
```

Comment:
Inline comments are moved to the next line and now describe the wrong line.
Those should either stay there or moved into the upper line.

Reply:
Oh, I just noticed now that there is already an open issue for this

Reply:
solved

Reply:
oh, no, it was not solved for patterns. Just list, attr-sets, let-ins